<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmarthomeWindow</class>
 <widget class="QMainWindow" name="SmarthomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f0f0f0; /* Светло-серый фон */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #f0f0f0; /* Светло-серый фон */
}</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>501</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLineEdit" name="loginLineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    padding: 8px; /* Внутренние отступы */
    border: 2px solid #bdc3c7; /* Серая рамка */
    border-radius: 8px; /* Закругленные углы */
    font-size: 14px; /* Размер шрифта */
    background-color: #ffffff; /* Белый фон */
    color: #2c3e50; /* Темно-синий текст */
}
QLineEdit:focus {
    border: 2px solid #3498db; /* Синяя рамка при фокусе */
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="registerButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4CAF50; /* Зеленый цвет */
    color: white; /* Белый текст */
    border-radius: 5px; /* Закругленные углы */
    padding: 10px; /* Внутренние отступы */
    font-size: 20px; /* Размер шрифта */
}
QPushButton:hover {
    background-color: #45a049; /* Цвет при наведении */
}</string>
       </property>
       <property name="text">
        <string>Зарегистрироваться</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    padding: 8px; /* Внутренние отступы */
    border: 2px solid #bdc3c7; /* Серая рамка */
    border-radius: 8px; /* Закругленные углы */
    font-size: 14px; /* Размер шрифта */
    background-color: #ffffff; /* Белый фон */
    color: #2c3e50; /* Темно-синий текст */
}
QLineEdit:focus {
    border: 2px solid #3498db; /* Синяя рамка при фокусе */
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-size: 22px; /* Увеличенный шрифт */
    color: #2c3e50; /* Темно-синий цвет */
}</string>
       </property>
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="loginButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4CAF50; /* Зеленый цвет */
    color: white; /* Белый текст */
    border-radius: 5px; /* Закругленные углы */
    padding: 10px; /* Внутренние отступы */
    font-size: 20px; /* Размер шрифта */
}
QPushButton:hover {
    background-color: #45a049; /* Цвет при наведении */
}</string>
       </property>
       <property name="text">
        <string>Вход</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="loginLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-size: 22px; /* Увеличенный шрифт */
    color: #2c3e50; /* Темно-синий цвет */
}</string>
       </property>
       <property name="text">
        <string>Логин:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 48px; /* Большой шрифт */
    font-weight: bold; /* Жирный текст */
    color: #3498db; /* Синий цвет */
    padding: 20px; /* Отступы */
}</string>
    </property>
    <property name="text">
     <string>Умный дом!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showPasswordCheckBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    font-size: 10px; /* Размер шрифта */
    color: #2c3e50; /* Темно-синий цвет */
}</string>
    </property>
    <property name="text">
     <string>Показать пароль</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
